Sources/OktaLogger/LoggerCore/OktaLoggerLogLevel.swift

```diff
@@ -32,14 +31,16 @@ public class OktaLoggerLogLevel: NSObject, OptionSet {
     @objc public static let info: OktaLoggerLogLevel = [OktaLoggerLogLevel(rawValue: 1 << 1), .warning]
     @objc public static let debug: OktaLoggerLogLevel = [OktaLoggerLogLevel(rawValue: 1 << 0), .info]
 
-    @objc public static let all: OktaLoggerLogLevel = [.debug, .info, .warning, .uiEvent, .error]
+    // Analytics
+    @objc public static let analytics: OktaLoggerLogLevel = [OktaLoggerLogLevel(rawValue: 1 << 5), .info]
```

Comment:
I think we are overloading Logger with extra functionality. Should Analytics be a separate framework?

Reply:
yup, separating the Analytics dependency